.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+
+## Prerequisites
+
+You must have a [GitHub account](https://help.github.com/en/github/getting-started-with-github/signing-up-for-a-new-github-account) with [two-factor authentication](https://help.github.com/en/github/authenticating-to-github/configuring-two-factor-authentication) enabled to contribute to Magento repositories. We also recommend creating a [personal access token](https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line) to use when interacting with GitHub in script from the the command line. 
+
+## Contribution requirements
+
+1. Contributions must adhere to the [Magento coding standards](https://devdocs.magento.com/guides/v2.3/coding-standards/bk-coding-standards.html).
+2. When you submit a Pull request (PR), write a meaningful description to explain the purpose of your contribution. Comprehensive descriptions increase the chances that a pull request can be merged quickly, without requests for additional clarification. See the [Magento Cloud Patches Pull Request Template](https://github.com/magento/magento-cloud-patches/blob/develop/.github/PULL_REQUEST_TEMPLATE.md) for more information.
+3. Commits must be accompanied by meaningful commit messages.
+4. If your PR includes bug fixes, provide a step-by-step description of how to reproduce the bug in the pull request description.
+3. If your PR includes new logic or new features, you must also submit the following information along with the pull request
+   * Unit/integration test coverage
+   * Proposed documentation updates: Submit developer documentation contributions to the [Magento DevDocs repository](https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md). Submit updates to Magento user documentation to the [Magento Merchant documentation repository](https://github.com/magento/merchdocs/blob/master/.github/CONTRIBUTING.md).
+4. For larger features or changes, [open an issue](https://github.com/magento/magento-cloud-patches/issues/new) to discuss the proposed changes prior to development. Discussing the updates in advance can prevent duplicate or unnecessary effort and allow other contributors to provide input.
+
+## Contribution process
+1. Search current [listed issues](https://github.com/magento/magento-cloud-patches/issues) (open or closed) for similar proposals of intended contribution before starting work on a new contribution.
+2. Review and sign the [Contributor License Agreement (CLA)](https://opensource.adobe.com/cla.html) if this is your first time contributing. You only need to sign the CLA once.
+3. Create and test your work.
+4. Fork the Magento Cloud repository according to the [Fork A Repository instructions](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#fork) and when you are ready to send us a pull request – follow the [Create A Pull Request instructions](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#pull_request).
```

Comment:
```suggestion
4. Fork the Magento Cloud Patches repository according to the [Fork A Repository instructions](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#fork) and when you are ready to send us a pull request – follow the [Create A Pull Request instructions](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#pull_request).
```

Reply:
The contributor guide and beginnner's guides no longer provide explicit instructions for submitting a pull request. Perhaps it would be better to just link to the GitHub help topic--submitting a pull request from a fork: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork